### PR TITLE
Improve continuous testing output

### PR DESCRIPTION
**Pull request by stuartwdouglas**
- Trim the console stack traces
- Add a 2 line summary at the bottom to make it easier to view the full results

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
While preparing for JNation I realised that the CT output as it currently stands involves a lot of console scrolling to see the details of the failures.

File: core/deployment/src/main/java/io/quarkus/deployment/dev/testing/JunitTestRunner.java [hunk 1]
```diff
@@ -8,18 +8,7 @@
...
-import java.util.Comparator;
-import java.util.Deque;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```
Comment (geoand) [inline core/deployment/src/main/java/io/quarkus/deployment/dev/testing/JunitTestRunner.java:11]: Can we get rid of these star import changes?

Comment (stuartwdouglas) [reply core/deployment/src/main/java/io/quarkus/deployment/dev/testing/JunitTestRunner.java]: oops, I had fixed it locally but forgot to push.

**Review by geoand — APPROVED** [review]

**Review by gsmet — APPROVED** [review]
I force pushed something to fix the failing test. The failure was related to the root path change.

**Pull request merged**